packages/falter-policyrouting/files/etc/init.d/freifunk-policyrouting

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh /etc/rc.common
 
 START=15
-. /lib/functions/network.sh
-. /lib/functions.sh
+. $IPKG_INSTROOT/lib/functions/network.sh
+. $IPKG_INSTROOT/lib/functions.sh
```

Comment:
Isnt that redundant to the line in /etc/rc.common include?

Reply:
The issue was about the warnings, and so naturally I didn't look at anything above or below.  Maybe we should do a general issue for code clean-up which would address things like that.